From: Roland Rosenfeld <roland@debian.org>
Subject: Stop removing MD5sums in clean target, because it the upstream
 source provides it and the file is missing after the cleanup otherwise.
Forwarded: Bjoern Jacke <bjoern@j3e.de> on Sat, 8 Sep 2007 13:06:08 +0200

--- a/Makefile
+++ b/Makefile
@@ -435,7 +435,7 @@ $(eval $(call liglist_template,liglist))
 ###
 
 clean:
-	rm -f all.words* igerman98*.tar.bz2 isowordlist* MD5sums SHA256sums
+	rm -f all.words* igerman98*.tar.bz2 isowordlist*
 	#sort liglist.aff  MD5sums myspell/*.dic de_DE.zip de_CH.zip myspell/VERSION *.tmp
 	rm -f ispell/*.aff ispell/*.hash
 	rm -f myspell/*.dic  myspell/*.aff  myspell/*.zip myspell/VERSION
